src/main/resources/swagger/api-docs.yaml

```diff
@@ -364,18 +364,26 @@ paths:
           default: false
         - in: body
           description: |
-            JSON object. Keys represent destinations, values represent sources.
-            Paths will be quoted, intermediate local directories made, and then sent to [gsutil cp](https://cloud.google.com/storage/docs/gsutil/commands/cp).
+            JSON object. Keys represent destinations, values represent sources. The following rules apply:
+
+            * If either the source or destination is a GCS path, it will be sent to [gsutil cp](https://cloud.google.com/storage/docs/gsutil/commands/cp). Therefore
+              this can be used to localize a file _from_ a bucket _to_ the notebook server; or delocalize a file _from_ the notebook server _to_ a bucket.
+            * If the source is a Data URI, then the destination file will be a created with the decoded data URI contents. This mode can only be used to localize
```

Comment:
Nit: remove _a_ in "will be _a_ created"

Reply:
good catch